src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -58,6 +58,12 @@ public async Task<ActionResult> Index(string hashedAccountId, string reservation
                 ViewBag.HideNav = true;
             }
 
+            var flashMessage = GetFlashMessageViewModelFromCookie();
+            if (flashMessage != null)
+            {
+                response.FlashMessage = flashMessage;
+            }
+
```

Comment:
this is being done in GetAccountInformation() called above

Reply:
Ah yes.  Will remove.